### PR TITLE
Modified dates

### PR DESCRIPTION
**Pull request by pzaichkina**
# Description

Aligning the recommendations on use of modified dates, and additional improvements to the pages:
/using-the-api/optimizing-api-calls
/using-the-api/modified-dates

## Type of change

Please delete options that are not relevant.

- [X] New document(s)/updating existing
- [ ] Fixes
- [ ] Styling
- [ ] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: docs/using-the-api/optimizing-api-calls.md [hunk 1]
```diff
@@ -9,44 +9,46 @@ Ensure you are familiar with our [rate limits](/using-the-api/rate-limits) befor
 
 :::
 
-Making as few calls to Codat as possible allows you to reduce your costs and delivers an improved user experience. With fewer calls, you process less data and access that data faster, meaning reduced waiting time for your users, less computational time for your systems, and decreased internet traffic between you and Codat.
+Making as few calls to Codat as possible allows you to reduce your costs and your system's computational times, and decreases internet traffic between you and Codat. It also allows you to process less data and access that data faster, delivering reduced waiting time and an improved user experience
```
Comment (mcclowes) [inline docs/using-the-api/optimizing-api-calls.md]:
```suggestion
Making as few calls to Codat as possible allows you to reduce your costs and your system's computational times, and decreases internet traffic between you and Codat. It also allows you to process less data and access that data faster, delivering reduced waiting time and an improved experience for your users.
```

File: docs/using-the-api/optimizing-api-calls.md [hunk 2]
```diff
@@ -9,44 +9,46 @@ Ensure you are familiar with our [rate limits](/using-the-api/rate-limits) befor
...
-We also recommend using a page size of 100 to return the results faster. This is especially relevant when a user is waiting for a UI to load. The maximum page size is 5000, which reduces the number of calls you make to our APIs, but provides you with a slower result.  
-
 If you also store the last queried date per company and per data type, you are able to call our API the minimum number of times to return only new or changed data.
 
 When a user links for the first time, there may be a need to pull all data, for all time. Make sure this is truly required, and if so, perform this pull only once. Afterwards, only query and access data since the last pull. 
 
-:::note Querying by the record date: example
+#### Pagination
+
+We also recommend using a page size of 100 to return the results faster. This is especially relevant when a user is waiting for a UI to load. The maximum page size is 5000, which reduces the number of calls you make to our APIs, but provides you with a slower result.
```
Comment (mcclowes) [inline docs/using-the-api/optimizing-api-calls.md:34]:
```suggestion
We also recommend using a page size of 100 to return the results faster. This is especially relevant when a user is waiting for a UI to load. The maximum page size is 5000 results, which means fewer API calls but much slower responses.
```

**Review by mcclowes — COMMENTED** [review]
Good to check the OAS descriptions for the dates stuff too

**Comment by pzaichkina** [comment]
> Good to check the OAS descriptions for the dates stuff too

Already done 🙃

**Review by mcclowes — APPROVED** [review]

**Pull request merged**